### PR TITLE
Update compilation docs

### PR DESCRIPTION
**Pull request by Hardcode84**

### CODE REVIEW

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -60,11 +60,25 @@ sorting values of a specific column:
...
+
+    if flag:
+       a = 1.0
+    else:
+       a = np.ones(10)
+     return a  # Type of a cannot be inferred
+
+:ref:`SDC API reference<apireference>`
+:ref:`More info on SDC compilation process and supported Python features<compilation>`
+`Numba documentation <https://numba.pydata.org/numba-doc/latest/user/troubleshoot.html>`_
```
Comment (PokhodenkoSA) [inline docs/source/getting_started.rst]: This links shows as undevided test. I propose to add `See also` and move them to separate string by inserting additional newline between them.

image

File: docs/source/getting_started.rst [hunk 2]
```diff
@@ -60,11 +60,25 @@ sorting values of a specific column:
 What If I Get A Compilation Error
 #################################
 
-.. todo::
-   Need to give basic information that hpat and numba do not support full set of Pandas and Numpy APIs, provide the link to the API Reference section for Intel® SDC, relevant reference to Numba documentation.
- 
-Also give very short introduction to what kind of code Numba/Intel® SDC can compile and what cannot, i.e. type stability etc. Provide the links to relevant sections in Intel® SDC and Numba documentations focusing on compilation issues/limitations
-  
+Not all Python code can be compiled with Intel® SDC. Not all Pandas and Numpy APIs are currently supported and not all valid python code can be compiled using underlying Numba compiler.
```
Comment (PokhodenkoSA) [inline docs/source/getting_started.rst]: Pandas and NumPy in other places are converted to links.
```suggestion
Not all Python code can be compiled with Intel® SDC. Not all `Pandas*`_ and `Numpy*`_ APIs are currently supported and not all valid python code can be compiled using underlying Numba compiler.
```

File: docs/source/getting_started.rst [hunk 3]
```diff
@@ -60,11 +60,25 @@ sorting values of a specific column:
...
+
+To be successfully compiled code must use only supported subset of Pandas API and use only subset Python supported by Numba (e.g. be type-stable)
```
Comment (PokhodenkoSA) [inline docs/source/getting_started.rst]:
```suggestion
To be successfully compiled code must use only supported subset of `Pandas*`_ API and use only subset Python supported by Numba (e.g. be type-stable)
```

**Comment by Hardcode84** [comment]
updated

**Pull request merged**